CHANGELOG.md

```diff
@@ -1,3 +1,118 @@
+## 1.3.0 (Unreleased)
+
+FEATURES:
+
+* **Native service discovery**: Register and discover services using builtin simple service discovery. [[GH-12368](https://github.com/hashicorp/nomad/issues/12368)]
+* core: Added support for reconnecting healthy allocations when disconnected clients reconnect. [[GH-12476](https://github.com/hashicorp/nomad/issues/12476)]
```

Comment:
```suggestion
* **Edge compute**: Added support for reconnecting healthy allocations when disconnected clients reconnect. [[GH-12476](https://github.com/hashicorp/nomad/issues/12476)]
```

Reply:
Ah yeah, this should've been done in the `.changelog` file. This is just merging it back, so we would want it to be the same. We can fix the file and have it look like this in the final release 👍